src/manager/evm/manager.rs

```diff
@@ -27,7 +28,8 @@ use crate::lotus::message::ipc::{QueryValidatorSetResponse, SubnetInfo, Validato
 use crate::lotus::message::wallet::WalletKeyType;
 use crate::manager::{EthManager, SubnetManager};
 
-pub type MiddlewareImpl = SignerMiddleware<Provider<Http>, Wallet<SigningKey>>;
+pub type NonSignerM = Provider<Http>;
```

Comment:
Are these used? If so let's use maybe `RawMiddleware` and `SignerMiddleware` so they are self-describing.

Reply:
We have two different middleware now, so it's easier for typing, I will update the naming accordingly, maybe we can have a better name than `SignerMiddleware` because it's taken by the `ethers` crate.